tools/SetupFlow/DevHome.SetupFlow/Models/RepositoryProvider.cs

```diff
@@ -71,13 +75,37 @@ public void StartIfNotRunning()
         Log.Logger?.ReportInfo(Log.Component.RepoConfig, "Starting DevId and Repository provider extensions");
         _devIdProvider = Task.Run(() => _extensionWrapper.GetProviderAsync<IDeveloperIdProvider>()).Result;
         _repositoryProvider = Task.Run(() => _extensionWrapper.GetProviderAsync<IRepositoryProvider>()).Result;
+
+        // _repositoryProvider2 == null is used to modify behavior.

```

Comment:
@EricJohnson327 Just wanted to make sure this is the right pattern. Especially with GetProviderAsync<IRepositoryProvider>() being used to get the object.